wildfly-standalone-centos7/scripts/wildfly-setup-redhat.sh

```diff
@@ -0,0 +1,67 @@
+#!/bin/sh
+
+echo "Red Hat WILDFLY 16.0.0.Final Standalone Intallation Start..." >> /home/$1/install.log
+/bin/date +%H:%M:%S  >> /home/$1/install.log
+
+export WILDFLY_USER=$2
+export WILDFLY_PASSWORD=$3
+
+echo "WILDFLY_USER: " ${WILDFLY_USER} >> /home/$1/install.log
+echo "WILDFLY_PASSWORD: " ${WILDFLY_PASSWORD} >> /home/$1/install.log
+
+echo "WILDFLY Downloading..." >> /home/$1/install.log
+cd /home/$1
+yum install -y git unzip java
+curl https://download.jboss.org/wildfly/16.0.0.Final/wildfly-16.0.0.Final.zip -o wildfly-16.0.0.Final.zip
+unzip wildfly-16.0.0.Final.zip
+
+echo "Sample app deploy..." >> /home/$1/install.log 
+git clone https://github.com/danieloh30/dukes.git
```

Comment:
you cannot clone a personal repo - you can add the files to the sample or use an official repo

Reply:
It doesn't make sense. There are templates to use a private git repo. For example, https://github.com/danieloh30/azure-quickstart-templates/blob/master/jboss-eap-standalone-rhel7/scripts/eap-setup-redhat.sh

What is the official repo? How do you define it?

Reply:
A non-personal repo - if there isn't one, add the files to this sample

Reply:
Check out the existing one.

https://github.com/Azure/azure-quickstart-templates/blob/master/jboss-eap-standalone-rhel7/scripts/eap-setup-redhat.sh#L42

Reply:
@bmoore-msft Any comments?